jobs/openshift/cluster/integreatly/test/Jenkinsfile

```diff
@@ -168,6 +164,8 @@ pipeline {
             [$class: 'BooleanParameterValue', name: 'dryRun', value: params.dryRun]
           ]
           build job: jobName, parameters: jobParams
+
+          notifyBuild(currentBuild.result, null)
```

Comment:
Not sure how to catch an error in this declarative pipeline. I'll need to take a look.